k8s/cmd/admission-webhook/const.go

```diff
@@ -12,6 +12,7 @@ const (
 	repoEnv              = "REPO"
 	initContainerEnv     = "INITCONTAINER"
 	tagEnv               = "TAG"
+	tracerEnabledEnv     = "TRACER_ENABLED"
```

Comment:
Shall we add these to `docs/env.md`?

Reply:
It is already at the top of docs/env.md ... but that said it defaulted to True.  Fixed that in latest push :)